skimage/segmentation/_quickshift.pyx

```diff
@@ -84,8 +84,8 @@ def quickshift(image, ratio=1.0, kernel_size=5, max_dist=10,
     # window size for neighboring pixels to consider
     if kernel_size < 1:
         raise ValueError("Sigma should be >= 1")
-    cdef int kernel_size_c = kernel_size
-    cdef int w = 3 * kernel_size_c
+    cdef float kernel_size_sq = kernel_size**2
+    cdef int w = int(3 * kernel_size)
```

Comment:
Won't matter too much, but cnp.ceil should be a wrapper around the ceil function in `<math.h>` and therefore faster in Cython code.